pinc/access_change_emails.inc

```diff
@@ -78,8 +80,15 @@ function f1_access_change_email($user, $access_change)
 
     $message[] = _("After all of the pages of a project are completed in F1, the project moves into F2 for review and any needed corrections. F2 is the final round of formatting before Post-Processing.");
 
-    $message[] = "***" . _("Please keep in mind that we have a 35 page-per-day limit in F1.") . "*** " .
-        _("(This is a daily limit, not an average.) The limit is there to make sure that we have a variety of projects available in the round at all times.");
+    if($daily_limit)
+    {
+        $message[] = "***" . sprintf( _("Please keep in mind that we currently have a limit in F1 of no more than %d pages per day."), $daily_limit) . "*** " .
+            _("(This is a daily limit, not an average.) The limit is there to make sure that we have a variety of projects available in the round at all times.");
+    // The above paragraph will be added to the email only if the daily limit
```

Comment:
Please indent this paragraph 4 more spaces so it aligns with the line above.

Reply:
Done.